Assets/MRTK/SDK/Experimental/Drawstring/Scripts/ElasticHandleLogic.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Physics.Experimental
+{
+    /// <summary>
+    /// Implements a two-handle elastic "stretch" logic, which allows for
+    /// either one or two pointers to stretch along a particular axis.
+    /// 
+    /// Built around the differential equations for a damped harmonic oscillator.
```

Comment:
If there's some interesting math or other reference (i.e. a wiki page) it can sometimes be useful to link to those to give others to reference those for future readers (i.e. the next question is what differential equations specfiically?)